src/graphics/gfx_print.cpp

```diff
@@ -98,60 +101,106 @@ int SuperTextPixLen(int SuperN, const char* SuperChars, int Font)
 
 void SuperPrintRightAlign(int SuperN, const char* SuperChars, int Font, float X, float Y, float r, float g, float b, float a)
 {
+    int RealFont = Font;
+    bool outline = false;
+
+    if(Font == 5)
+    {
+        Font = 4;
+        outline = true;
+    }
+
     int dFont = NewFontRender ? FontManager::fontIdFromSmbxFont(Font) : -1;
 
     if(dFont >= 0)
     {
         X -= FontManager::textSize(SuperChars, SuperN, dFont, 0, false, FontManager::fontSizeFromSmbxFont(Font)).w();
-        FontManager::printText(SuperChars, SuperN, X, Y, dFont, r, g, b, a, FontManager::fontSizeFromSmbxFont(Font));
+        FontManager::printText(SuperChars, SuperN, X, Y, dFont, r, g, b, a, FontManager::fontSizeFromSmbxFont(Font), outline);
         return;
     }
 
     X -= SuperTextPixLen(SuperN, SuperChars, Font);
-    SuperPrint(SuperN, SuperChars, Font, X, Y, r, g, b, a);
+    SuperPrint(SuperN, SuperChars, RealFont, X, Y, r, g, b, a);
 }
 
 void SuperPrintCenter(int SuperN, const char* SuperChars, int Font, float X, float Y, float r, float g, float b, float a)
 {
+    int RealFont = Font;
+    bool outline = false;
+
+    if(Font == 5)
+    {
+        Font = 4;
+        outline = true;
+    }
+
     int dFont = NewFontRender ? FontManager::fontIdFromSmbxFont(Font) : -1;
 
     if(dFont >= 0)
     {
         X -= FontManager::textSize(SuperChars, SuperN, dFont, 0, false, FontManager::fontSizeFromSmbxFont(Font)).w() / 2;
-        FontManager::printText(SuperChars, SuperN, X, Y, dFont, r, g, b, a, FontManager::fontSizeFromSmbxFont(Font));
+        FontManager::printText(SuperChars, SuperN, X, Y, dFont, r, g, b, a, FontManager::fontSizeFromSmbxFont(Font), outline);
         return;
     }
 
     X -= SuperTextPixLen(SuperN, SuperChars, Font) / 2;
-    SuperPrint(SuperN, SuperChars, Font, X, Y, r, g, b, a);
+    SuperPrint(SuperN, SuperChars, RealFont, X, Y, r, g, b, a);
 }
 
 void SuperPrintScreenCenter(int SuperN, const char* SuperChars, int Font, float Y, float r, float g, float b, float a)
 {
+    int RealFont = Font;
+    bool outline = false;
+
+    if(Font == 5)
+    {
+        Font = 4;
+        outline = true;
+    }
+
+    if(Font == 5)
+        Font = 4;
```

Comment:
Are you reserving the font 5 for internal needs?

Reply:
Yes, that's right. Let me see... do you know if there is a way to check whether there is a user-specified Font 5? It would be nice to make this hardcoded re-use only occur if the default Font 5 would be used.